h2o-py/build.gradle

```diff
@@ -37,13 +37,37 @@ task verifyDependencies(type: Exec) {
                                           "--metayaml", "conda/h2o/meta.yaml"])
 }
 
+task copyH2OJar(type: Copy) {
+    from "${rootDir}/build/h2o.jar"
+    into "${buildDir}/h2o/backend/bin/"
+}
+
+task copySrcFiles(type: Copy, dependsOn: [copyH2OJar]) {
+    from ("${projectDir}") {
+        include "setup.py"
+        include "setup.cfg"
+        include "h2o/**"
+        include "DESCRIPTION.rst"
+        include "MANIFEST.in"
+        include "README.md"
+        include "requirements.txt"
+    }
+    into "${buildDir}"
+}
 
-task buildDist(type: Exec, dependsOn: [verifyDependencies, createVersionFiles]) {
+task buildDist(type: Exec, dependsOn: [verifyDependencies, createVersionFiles, copySrcFiles]) {
+    workingDir buildDir
     doFirst {
         file("${buildDir}/tmp").mkdirs()
         standardOutput = new FileOutputStream(file("${buildDir}/tmp/h2o-py_buildDist.out"))
     }
     commandLine getOsSpecificCommandLine([pythonexe, "setup.py", "bdist_wheel"])
+
+    file("${projectDir}/dist/").mkdirs()
+    copy {
+        from "${buildDir}/dist/*.whl"
+        into "${projectDir}/dist/"
```

Comment:
This breaks what we want to achieve here. All generated artifacts should be stored in disposable directory `build`.

I would just keep the distribution in `build/dist` and would not copy it anywhere else. Having everything in build makes the clean work properly

Reply:
Should be removed, thanks @jakubhava.